roles/preseed_site/files/preseed/firstBootClient.sh

```diff
@@ -17,7 +17,7 @@ xbps-install -R http://repo.collegiumv.org/current -y curl fbv ansible git-all p
 pip install netaddr
 
 # Attempt to run the main ansible installer
-ansible-pull -U https://github.com/collegiumv/cv_config.git
+ansible-pull -U https://github.com/collegiumv/cv_config.git workstation.yml
```

Comment:
local.yml is a hardcoded lookup which ansible-pull will search for in the absence of any other playbook.  It is however less intuitive than a named playbook.